tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -179,15 +155,15 @@ def test_universal_autolog_attaches_pyspark_import_hook_if_pyspark_isnt_installe
         autolog_mock.side_effect = ImportError("no module named pyspark blahblah")
 
         mlflow.autolog(**config)
-        autolog_mock.assert_called_once()  # it was called once and failed
+        autolog_mock.assert_not_called()
 
         # now the user installs pyspark
         autolog_mock.side_effect = None
 
         mlflow.utils.import_hooks.notify_module_loaded(pyspark)
 
         # assert autolog is called again once pyspark is imported
-        assert autolog_mock.call_count == 2
+        assert autolog_mock.call_count == 1
```

Comment:
I think we need to fix the comment above. `again` indicates `call_count` should be 2, but it's actually 1.